ansible/group_vars/all

```diff
@@ -37,13 +37,14 @@ application_name: "pouta_blueprints"
 application_user: "{{ application_name }}"
 application_group: "webapps"
 application_debug_logging: True
-application_secret_key: "change me"
+application_secret_key: "change_me"
 application_config_dir: "/etc/{{ application_name }}"
 application_config: "{{ application_config_dir }}/config.yaml"
 application_database_name: "{{ application_name }}"
 application_database_user: postgres
 application_database_password: "{{ application_secret_key }}"
 application_database_uri: "postgresql://postgres:{{ application_database_password }}@db/{{ application_database_name }}"
+application_database_uri_devel: "postgresql://postgres:{{ application_database_password }}@{{ domain }}/{{ application_database_name }}"
```

Comment:
I'd rather use same variable for both deployments. db can be an alias in /etc/hosts when deploying with vagrant